### PR TITLE
Feat/textarea check

### PR DESCRIPTION
**Pull request by andresz1**
## Textarea update

**TASK**: #930 

### Description, Motivation and Context
This PR:
- Update `Textarea` component introducing `TextareaGroup`
- Update `InputGroup` component to use `id` instead of `displayName` to render children. This decouples the need of using input display names
- Update documentation

Notes:
- Once this PR is merged I think `Textarea` could be ready to be used
- Test related with form field in the text area does not work 😞 we are checking why

### Types of changes
- [ ] 🛠️ Tool
- [ ] 🪲 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles

### CODE REVIEW

File: packages/components/input/src/InputClearButton.tsx [hunk 1]
```diff
@@ -47,4 +47,8 @@ export const InputClearButton = forwardRef<HTMLButtonElement, InputClearButtonPr
   }
 )
 
+export const InputClearButton = Object.assign(Root, {
+  id: 'ClearButton',
+})
```
Comment (Powerplex) [inline packages/components/input/src/InputClearButton.tsx:52]: I'm still not familiar with assigning an `id` to the component like this instead of a displayName. Can you summarize it please ? :)

**Review by Powerplex — APPROVED** [review]

**Review by kikoruiz — APPROVED** [review]

File: packages/components/input/src/InputClearButton.tsx (hunk 1 above)
Comment (andresz1) [reply packages/components/input/src/InputClearButton.tsx:52]: Yes. I did this change to not be coupled with the `displayName`. For example, to add the `Textarea` I would have needed to go to the `InputGroup` and add support for it
```tsx
const input = findElement('Input', 'Textarea')
```
Doing it this way no changes need to be performed in the `InputGroup` and consumer can also created their own component that can work nicely with the `InputGroup` is needed extending the `id`.

I was inspired by chakra [InputGroup](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/input/src/input-group.tsx#L44) to perform this change
